### PR TITLE
Tracks: Fix CodeSniffer errors

### PR DESCRIPTION
**Pull request by psealock**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

phpcs fixes for touched files in Tracks project.

### CODE REVIEW

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 1]
```diff
@@ -536,7 +536,7 @@ class="location-input wc-enhanced-select dropdown"
 	public function wc_setup_store_setup_save() {
 		check_admin_referer( 'wc-setup' );
 
-		// phpcs:disable WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.VIP.ValidatedSanitizedInput.InputNotValidated, WordPress.VIP.ValidatedSanitizedInput.MissingUnslash
+		// phpcs:disable WordPress.VIP.SuperGlobalInputUsage.AccessDetected, WordPress.VIP.ValidatedSanitizedInput.InputNotValidated, WordPress.VIP.ValidatedSanitizedInput.MissingUnslash, WordPress.Security.ValidatedSanitizedInput
```
Comment (mikejolley) [inline includes/admin/class-wc-admin-setup-wizard.php]: I've just noticed all those disables... Maybe this block should be checked again. For example, unslashing is needed!

e.g.:

```
$address        = sanitize_text_field( $_POST['store_address'] );
```

to:

```
$address        = wc_clean( wp_unslash( $_POST['store_address'] ) );
```

Would you mind checking this? Also bare in mind the .VIP. rules no longer exist. I think it's .SECURITY. so they can be removed.

File: includes/admin/class-wc-admin.php [hunk 2]
```diff
@@ -203,29 +207,29 @@ public function prevent_admin_access() {
...
 			$email = new WC_Email();
 
-			// wrap the content with the email template and then add styles
+			// wrap the content with the email template and then add styles.
 			$message = apply_filters( 'woocommerce_mail_content', $email->style_inline( $mailer->wrap_message( $email_heading, $message ) ) );
 
-			// print the preview email
-			echo $message;
+			// print the preview email.
+			echo esc_html( $message );
```
Comment (mikejolley) [inline includes/admin/class-wc-admin.php]: Might be html in here. Probably best to echo and add exclusion rule. It's not user generated.

**Review by mikejolley — CHANGES_REQUESTED** [review]
Added some comments. The disables in wizard need a look.

**Comment by psealock** [comment]
Thanks @mikejolley !

I updated the preview emails `$message` to not escape html and removed the wizard disables in favour of `wc_clean` and `wp_unslash`. Thanks for the tip.

**Review by mikejolley — APPROVED** [review]

**Pull request merged**